### ISSUE TITLE
[Suggestion] Allow using per-instance config for only some instances, and global config for others

### ISSUE THREAD

**Issue by unilock** [emilyploszaj/emi#279]
I tend to run many different Minecraft versions when testing mods, and typically include EMI in those test instances for convenience.

It would be nice if I could use the global EMI config for those test instances (+ other instances), but use a per-instance EMI config for other instances that require notable changes - for example, in a modpack that includes FTB Quests, I've increased the top margin of the left sidebar to make room for the "Quests" button; a change that doesn't make sense on other instances which don't have such a button.

Currently, I can either use the global config for all instances of EMI, or use a per-instance config for every instance at once.

**Comment by emilyploszaj** [comment]
Yeah this is a long term goal, especially as the dev I want my real global config for playing but I want my dev environments to use local files, I just haven't found the time for it yet, will look into it.
